services/mailer/mail.go

```diff
@@ -46,6 +47,30 @@ var (
 	subjectRemoveSpaces = regexp.MustCompile(`[\s]+`)
 )
 
+type MailRecipient struct {
```

Comment:
not sure how mouch mem we save with this ... second option is to just use the raw models.User struct